src/sql/workbench/parts/query/browser/queryResultsView.ts

```diff
@@ -326,12 +397,27 @@ export class QueryResultsView extends Disposable {
 		this.chartTab.chart(dataId);
 	}
 
+	public sandDanceData(dataId: { resultId: number, batchId: number }): void {
+		this.input.state.visibleTabs.add(this.sandDanceTab.identifier);
```

Comment:
Would prefer that this and the other similar methods are named something like showSandDanceTab or showDataAsSandDance. It's more descriptive of what the function is actually doing.